specification/azuredatatransfer/resource-manager/Microsoft.AzureDataTransfer/preview/2023-10-11-preview/azuredatatransfer.json

```diff
@@ -0,0 +1,2304 @@
+        },
+        "operationId": "Pipelines_ListBySubscription",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Listed all pipelines under the subscription.",
+            "schema": {
+              "$ref": "#/definitions/pipelinesListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.AzureDataTransfer/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Operations_List",
+        "x-ms-examples": {
+          "List operations": {
+            "$ref": "./examples/listOperations.json"
+          }
+        },
+        "description": "Lists all of the available Microsoft.AzureDataTransfer REST API operations.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Listed all of the available operations.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "actionRequest": {
+      "type": "object",
+      "description": "An action request",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the action"
+        },
+        "payload": {
+          "type": "object",
+          "description": "Action request payload",
+          "additionalProperties": {}
+        }
+      }
+    },
+    "actionResult": {
+      "type": "object",
+      "description": "An action result",
+      "properties": {
+        "result": {
+          "type": "object",
+          "description": "Action request result",
+          "additionalProperties": {}
+        }
+      }
+    },
+    "selectedResource": {
```

Comment:
>selectedResource [](http://example.com/codeflow?start=5&length=16)

This looks a lot like TrackedResource in common-types but not quite the same. Why the differences?

Reply:
https://learn.microsoft.com/en-us/azure/azure-resource-manager/managed-applications/microsoft-solutions-resourceselector

This maps to the output of the ResourceSelector UI element which is not the same as the TrackedResource.